src/app/modules/vault-filter/components/status-filter.component.html

```diff
@@ -0,0 +1,34 @@
+<ng-container *ngIf="show">
+  <h2 class="sr-only">{{ "filters" | i18n }}</h2>
```

Comment:
This won't be read out by screenreaders because it's not focusable/selectable. I'm not sure what the solution is.

Reply:
This was already an `h2` tag and already had this class, so whatever is going on here we aren't regressing with these changes.